socketcan_bridge/src/socketcan_to_topic.cpp

```diff
@@ -48,7 +48,9 @@ namespace socketcan_bridge
   SocketCANToTopic::SocketCANToTopic(ros::NodeHandle* nh, ros::NodeHandle* nh_param,
       can::DriverInterfaceSharedPtr driver)
     {
-      can_topic_ = nh->advertise<can_msgs::Frame>("received_messages", 10);
+      int queue_size;
+      nh_param->param<int>("received_messages_queue_size", queue_size, 10);
+      can_topic_ = nh->advertise<can_msgs::Frame>("received_messages", queue_size);
```

Comment:
Why not just
```
      can_topic_ = nh->advertise<can_msgs::Frame>("received_messages", nh_param->param("received_messages_queue_size", 10));
```

Reply:
I honestly was not aware that overload of <code>NodeHandle::param</code> existed and just used what I have been used to